LadybugTools_Adapter/AdapterActions/Push.cs

```diff
@@ -0,0 +1,28 @@
+﻿using BH.Engine.Adapter;
+using BH.Engine.LadybugTools;
+using BH.oM.Adapter;
+using BH.oM.Base;
+using BH.oM.Data.Requests;
+using BH.oM.LadybugTools;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace BH.Adapter.LadybugTools
+{
+    public partial class LadybugToolsAdapter : BHoMAdapter
+    {
+        public override List<object> Push(IEnumerable<object> objects, string tag = "", PushType pushType = PushType.AdapterDefault, ActionConfig actionConfig = null)
+        {
+            if (actionConfig is null)
```

Comment:
```suggestion
            if (actionConfig == null)
```

You're not in Python land here @Tom-Kingstone 😉 

I would also recommend trying to cast the config to the LadybugTools_Toolkit config and checking that can be done successfully and returning an error here if it's not, before you get to the `CreateLadybug` line.